site/docs/es/plugins/session.md

```diff
@@ -669,7 +669,7 @@ En la práctica, en lugar de tener los datos de la sesión disponibles en `ctx.s
 
 ```ts
 // Sesiones por defecto (sesiones estrictas)
-bot.command("settings", (ctx) => {
+bot.command("settings", async (ctx) => {
```

Comment:
Why is the highlighting broken here, is there no parse?

Reply:
Not sure, but the file looks correct locally. Seems like just an highlighting issue on Github's side.